modules/upgrade/pages/k-compatibility.adoc

```diff
@@ -1,53 +1,85 @@
+The Redpanda Helm chart and Redpanda Operator are versioned and tested alongside specific Redpanda core releases, Kubernetes, and Helm.
 
-[[rp]]
-== Redpanda Helm chart
+Starting from version v25.1-k8s, the Redpanda Operator and Redpanda Helm chart follow a new versioning scheme aligned with Redpanda core releases:
 
-The Redpanda Helm chart is compatible with these versions of Redpanda, Kubernetes, and Helm.
+- `v25.1` refers to the Redpanda core feature release deployed by default.
 
-NOTE: The minimum recommended version of Kubernetes is 1.25.
+- The suffix `-k8s<number>` denotes the Kubernetes-specific version, where `<number>` is the unique patch version for either the operator or Helm chart.
 
-include::upgrade:partial$k-redpanda-compatibility-matrix.adoc[]
+Each Redpanda Operator and Helm chart version supports the corresponding Redpanda core version plus one minor version above and one below. This approach ensures flexibility and ease during upgrades. For example, Redpanda Operator version 25.1-k8sx supports Redpanda core versions 25.2.x, 25.1.x, and 24.3.x.
 
-By default, the Redpanda Helm chart depends on cert-manager for enabling TLS.
-Make sure to install a version of the cert-manager CRDs that supports the `certificates.cert-manager.io/v1` resource.
-See the https://github.com/cert-manager/cert-manager/releases[cert-manager release notes].
+Redpanda Operator and Helm chart versions are supported only while their associated Redpanda core version remains supported. If the core version reaches End-of-Life (EoL), the corresponding Operator and Helm chart versions also reach EoL.
 
-== Redpanda Operator
+NOTE: Beta versions are available only for testing and feedback. They are not supported by Redpanda and must not be used in production environments. To give feedback on beta releases, reach out to the Redpanda team in https://redpanda.com/slack[Redpanda Community Slack^].
 
-Each version of the Redpanda Operator and its corresponding set of custom resource definitions (CRDs) supports specific versions of the Redpanda Helm chart.
+== Compatibility matrix
 
-Although new releases of the Redpanda Operator are backward compatible with existing versions of the Redpanda Helm chart, new releases of the Redpanda Helm chart might introduce changes to the Helm values, which could require you to upgrade both the Redpanda Operator and the accompanying CRDs.
+[cols="1a,1a,1a,1a,1a,1a"]
+|===
+|Redpanda Core / `rpk` |Helm Chart |Operator Helm Chart |Operator |Helm CLI |Kubernetes
 
-Consider the following best practices for upgrades:
+// |25.2.x
+// |25.3-k8sx, 25.2-k8sx, 25.1-k8sx
+// |25.3-k8sx, 25.2-k8sx, 25.1-k8sx
+// |25.3-k8sx, 25.2-k8sx, 25.1-k8sx
+// |3.13+
+// |1.30.x - 1.33.x
 
-- *Prioritize updating the Redpanda Operator*: Always aim to use the latest version of the Redpanda Operator.
+|25.1.x
+|25.1-k8s-beta*, 5.10.x, 5.9.x
```

Comment:
What does the asterisks here indicate? The way it renders, it looks like there should be a foot note below the chart but there isn't.

<img width="1024" alt="Screenshot 2025-04-03 at 10 14 51" src="https://github.com/user-attachments/assets/caf78380-6d6b-4919-89bd-e4bd475fa72e" />